preupg/creator/settings.py

```diff
@@ -43,7 +43,7 @@
 properties_ini = "- Fill empty options in %s before you use it."
 prop_src_version = "Specify major source OS version e.g. \"6\":"
 prop_dst_version = "Specify major destination OS version e.g. \"7\":"
-
-text_for_testing = "\nFor testing content run these two commands:\n" \
-                   "- preupg-create-group-xml %s to create XML file\n" \
-                   "- preupg -c %s"
```

Comment:
now result message is:
`preupg -c modules-results/system/packages/all-xccdf.xml`
but needs to be:
`preupg -c modules-results/all-xccdf.xml`

Reply:
Thanks. Fixed.